src/superRare.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  WhitelistCreator,
+  Bid,
+  AcceptBid,
+  CancelBid,
+  Sold,
+  SalePriceSet,
+  OwnershipTransferred,
+  Transfer,
+  Approval,
+  SuperRare,
+} from "../generated/SuperRare/SuperRare";
+import { Nft, NftContract } from "../generated/schema";
+import { ZERO_ADDRESS } from "./constants";
+import { getOrCreateAccount } from "./utils";
+
+/**
+ * Standard ERC721
+ */
+export function handleTransfer(event: Transfer): void {
+  let contract = SuperRare.bind(event.address);
+  let address = event.address.toHexString();
+  let tokenId = address + "/" + event.params._tokenId.toString();
+  let account = getOrCreateAccount(event.params._to);
+
+  if (NftContract.load(address) == null) {
+    let nftContract = new NftContract(address);
+    nftContract.name = contract.name();
+    nftContract.symbol = contract.symbol();
+    nftContract.platform = "SuperRare";
+    nftContract.save();
+  }
+
+  let nft = Nft.load(tokenId);
+  if (nft == null) {
+    nft = new Nft(tokenId);
+    nft.contract = address;
+    nft.tokenID = event.params._tokenId;
+    nft.tokenURI = contract.tokenURI(event.params._tokenId);
+    nft.createdAt = event.block.timestamp;
+  }
+  if (event.params._to.toHex() == ZERO_ADDRESS) {
+    // burn token
+    nft.removedAt = event.block.timestamp;
+  }
+  nft.owner = account.id;
+  nft.save();
+}
+
+export function handleApproval(event: Approval): void { }
+
+/**
+ * SuperRare specific
+ */
+export function handleWhitelistCreator(event: WhitelistCreator): void {
+  let account = getOrCreateAccount(event.params._creator);
+  account.isCreator = true;
```

Comment:
Hm, are you sure that this property `isCreator` is something that can be interpolated to other platforms?

Reply:
That is a good question. I mean if 9 out of 10 platforms have it, do we use it or not? Do we take the standard and expand it or do we just stick to the standard?

Reply:
I would just create an array of NTFs created - it seems like more useful info.
And if it's not empty, you can assume that the person is a creator